manim/scene/scene_file_writer.py

```diff
@@ -421,6 +426,28 @@ def close_movie_pipe(self):
         shutil.move(
             self.temp_partial_movie_file_path, self.partial_movie_file_path,
         )
+        logger.debug(
+            f"Animation {self.scene.num_plays} : Partial movie file written in {self.partial_movie_file_path}"
+        )
+
+    def is_already_cached(self, hash_invokation):
```

Comment:
Nitpick: the correct spelling is `invocation` :)

Reply:
Thank you !

_(this is slightly embarrassing)_